appcues/src/main/java/com/appcues/trait/appcues/BackdropTrait.kt

```diff
@@ -27,6 +28,10 @@ internal class BackdropTrait(
         const val TYPE = "@appcues/backdrop"
     }
 
+    override val type = TYPE
+
+    override val level = ExperienceTraitLevel.GROUP
```

Comment:
And here is where I thought that the TraitMapper would pass in the level that the trait was being applied to, at construction time, rather than hardcoding any particular trait to any particular level.   i.e. perhaps some future use case would allow a backdrop at the Experience level here (no idea what that use case is, just an example) - it would all be seamlessly passed through during the experience mapping time.  

Most Traits would not need it at all, seemingly - only the new background-content.  Maybe it is not even part of the public `ExperienceTrait` interface - just a constructor arg for the one that needs it.

Reply:
the problem is that I wanted to check for something relatable to ExperienceTrait, so we can use that in a for loop in MapperExt

Reply:
one thing we could do is convert some interfaces to abstract class, then we can set this one just once in the super class